etc/desimodel_sync_kpno_cron.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Path to the sync directory, containing the desimodel data trunk
+# checkout and logs.
+syncdir=/n/home/datasystems/desimodel_sync
+logdir="${syncdir}/logs"
+svntrunk="${syncdir}/desimodel_trunk"
```

Comment:
For the record: I like that you are using a separate checkout of desimodel/data for this purpose, which seems more robust than pigging backing on some other installed checkout.